pipelines/streaming/src/main/java/org/openmrs/analytics/DebeziumListener.java

```diff
@@ -116,19 +116,19 @@ private String getDebeziumConfig() {
         + debeziumConfigs.get("databaseServerName")
         + "?"
         + "databaseHostname="
-        + debeziumConfigs.get("databaseHostName")
+        + databaseConfiguration.getDatabaseHostName()
```

Comment:
Is it cleaner to have this built via `String.format`?

Reply:
`String.format` will make this harder to read IMO; I can change it to `StringBuilder` if your concern is performance but this function is called only on initialization.

Reply:
No worries, this works!